app/src/main/java/com/lambdaschool/hackathon_portal/retrofit/HackathonApiInterface.kt

```diff
@@ -26,4 +26,19 @@ interface HackathonApiInterface {
     @DELETE("users/{id}")
     fun deleteUser(@Path("id") id: Int?,
                    @Header("Authorization") bearerToken: String): Call<Deletion>
+
+    @GET("hackathons/{id}")
+    fun getHackathon(@Path("id") id: Int,
+                     @Header("Authorization") bearerToken: String): Call<Hackathon>
+
+    @PUT("hackathons/{hack_id}/u/{org_id}")
+    fun updateHackathon(@Path("hack_id") hack_id: Int,
+                        @Path("org_id") org_id: Int,
+                        @Header("Authorization") bearerToken: String,
+                        @Body hackathon: Hackathon): Call<Hackathon>
```

Comment:
Fine for now, but will need to change later to use a JsonObject that is built on the fragment of only the fields needing to be updated. Maybe add a //TODO here?
```suggestion
                        // TODO: use a JsonObject of only the fields that need to be updated which
                        //  gets built on the fragment before making the method call.
                        @Body hackathon: Hackathon): Call<Hackathon>
```

Reply:
Added in commit 0ccde73ddeb7d91f9fefbcea8e4c1822d3ba3025